theano/tensor/tests/test_basic.py

```diff
@@ -1650,7 +1650,7 @@ def test_py_c_match():
 _eps = 1e-10
 _good_broadcast_unary_arctanh = dict(
     normal=(rand_ranged(-1 + _eps, 1 - _eps, (2, 3)),),
-    integers=(randint_ranged(-1 + _eps, 1 - _eps, (2, 3)),),
+    integers=(randint_ranged(-1, 1, (2, 3)),),
```

Comment:
arctanh is not defined at -1 or 1, so the only integer-valued point where it makes sense is 0.
The earlier code would basically be equivalent to `random_integers(0, 0, (2, 3))` or `randint(0, 1, (2, 3))`.
Just use zeros(2, 3) I guess...